php/.bashrc

```diff
@@ -0,0 +1,2 @@
+PS1="\`if [ \$? = 0 ]; then echo \[\e[37m\]^_^[\u@\H:\w]\\$ \[\e[0m\]; else echo \[\e[31m\]O_O[\u@\H:\w]\\$ \[\e[0m\]; fi\`"
```

Comment:
`[\u@\H:\w]`
Instead of this above part of the prompt, we can have
`[\u@$VIRTUAL_HOST:\w]` as the hostname for containers.

That way we'll have a prompt like
`^_^[root@example.com:/var/www/html]`